apiary.apib

```diff
@@ -3,34 +3,99 @@ HOST: http://api.superdesk.org/v1/
 
 # Superdesk Content API
 
-The Superdesk Content Delivery API is an Open API. It aims to empower news
-organisations using Superdesk to open up their content to a broad community of
+The Superdesk Content Delivery API is a public API. It aims to empower news
+organizations using Superdesk to open up their content to a broad community of
 innovative developers and encourage the development of a diversity of news
-applications.
+applications to engage readers wherever they are.
 
 # Current Version
 
-TODO
+We are currently developing the __Superdesk Content API v1__ version, the release
+date is yet to be announced.
 
-# Content Schema
+When working with the __Superdesk Content API__, you must explicitly request the
+version of the API you want to use via URL, as in the following request:
 
-TODO
+```
+https://api.superdesk.org/v1/items
+```
+
+# Content Model
+
+Content in Superdesk is organized following the __News Architecture__ schema defined as
+part of the [NewsML-G2](https://iptc.org/standards/newsml-g2/) standard for exchanging
+news and news management.
```

Comment:
Here we should specify the format we use (ninjs) and the fact that it's a flattened NewsML2